cmake/stm32/utilities.cmake

```
@@ -11,19 +11,32 @@ function(stm32_util_create_family_targets FAMILY)
         message(FATAL_ERROR "Expected at most one core for family ${FAMILY}: ${CORES}")
     endif()
 
+    option(STM32_ENABLE_FAST_MATH "Add -ffast-math compiler flag" OFF)
+    option(STM32_NO_BUILTIN "Add -fno-builtin compiler flag" OFF)
+    option(STM32_NO_STRICT_ALIASING "Add -fno-strict-aliasing compiler flag" OFF)
+
     if(NOT (TARGET STM32::${FAMILY}${CORE_C}))
+        set(STM32_COMPILE_OPTIONS
+            -mthumb -mabi=aapcs -Wall -ffunction-sections -fdata-sections

```

I'm usure about what does `-mabi=aapcs` do ?
I never used it before maybe it could be removed also
```suggestion
            -mthumb -Wall -ffunction-sections -fdata-sections
```